app/src/storage/update-jenkins-server-state.ts

```diff
@@ -22,21 +23,26 @@ export const updatePipelines = (
 	}
 };
 
+async function getJenkinsServer(uuid: string, logger?: Logger): Promise<JenkinsServer> {
```

Comment:
This could be hoisted oot to a utils but not yet

Reply:
But not yet... love the vagueness 😛